aws-distro-opentelemetry-node-autoinstrumentation/test/patches/instrumentation-patch.test.ts

```diff
@@ -124,6 +153,75 @@ describe('InstrumentationPatchTest', () => {
+    const bedrockAttributesAfterResponse: Attributes = doResponseHookBedrock(services, 'Bedrock');
+    expect(bedrockAttributesAfterResponse[AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_GUARDRAIL_ID]).toEqual(_BEDROCK_GUARDRAIL_ID);
+  });
+
+  it('Bedrock Agent with patching', () => {
+    const patchedAwsSdkInstrumentation: AwsInstrumentation = extractAwsSdkInstrumentation(PATCHED_INSTRUMENTATIONS);
+    const services: Map<string, any> = extractServicesFromAwsSdkInstrumentation(patchedAwsSdkInstrumentation);
+
+    const operation_to_expected_attribute: Object = {
+      CreateAgentActionGroup: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      CreateAgentAlias: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      DeleteAgentActionGroup: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      DeleteAgentAlias: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      DeleteAgent: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      DeleteAgentVersion: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      GetAgentActionGroup: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      GetAgentAlias: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      GetAgent: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      GetAgentVersion: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      ListAgentActionGroups: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      ListAgentAliases: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      ListAgentKnowledgeBases: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      ListAgentVersions: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      PrepareAgent: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      UpdateAgentActionGroup: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      UpdateAgentAlias: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      UpdateAgent: { 'aws.bedrock.agent.id': _BEDROCK_AGENT_ID },
+      AssociateAgentKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      CreateDataSource: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      DeleteKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      DisassociateAgentKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      GetAgentKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      GetKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      ListDataSources: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      UpdateAgentKnowledgeBase: { 'aws.bedrock.knowledge_base.id': _BEDROCK_KNOWLEDGEBASE_ID },
+      DeleteDataSource: { 'aws.bedrock.data_source.id': _BEDROCK_DATASOURCE_ID },
+      GetDataSource: { 'aws.bedrock.data_source.id': _BEDROCK_DATASOURCE_ID },
+      UpdateDataSource: { 'aws.bedrock.data_source.id': _BEDROCK_DATASOURCE_ID },
+    };
+
+    for (const [operation, attribute_tuple] of Object.entries(operation_to_expected_attribute)) {
```

Comment:
This seems to be missing the equivalent of the Python test case:
```
            bedrock_agent_success_attributes: Dict[str, str] = _do_on_success_bedrock("bedrock-agent", operation)
            self.assertEqual(len(bedrock_agent_success_attributes), 1)
            self.assertEqual(bedrock_agent_success_attributes[attribute_tuple[0]], attribute_tuple[1])
```
https://github.com/aws-observability/aws-otel-python-instrumentation/pull/209/files#diff-ec461b6261efcd381c9fa4b29a296bb17143126ad6d3c1229a02b253c7ed4077